tabcmd/commands/commands.py

```diff
@@ -13,13 +13,6 @@ def __init__(self, args):
         self.personal_token = args.token
         self.logging_level = args.logging_level
 
-    @staticmethod
-    def exit_with_error(logger, error_message, exception=None):
```

Comment:
was this all done by an automated tool? This method is used in a couple of places, for instance in commands/auth/session.py.

Reply:
Yea this is all from flake8

Reply:
ah, I missed that the method was defined twice, thought it was being deleted altogether.